### PR TITLE
Update readme to template changes

### LINKED ISSUE #77: Update repo ReadMe.md file
In line with the issue available [here](https://github.com/BHoM/admin/issues/4), please update your repositories ReadMe.md file to match the template provided in the linked admin issue.

Please remember to change any instances of `Robot_Toolkit` in the template to your repository name.

I will comment separately with a specific AppVeyor badge link that should be used for this repo.

When you resolve this issue, please add myself and @al-fisher as reviewers.

### PR DESCRIPTION
**Pull request by MartinHenriksen**
### Issues addressed by this PR

 Closes #77

 
readme.md has been updated to follow latest template changes

 ### Changelog

 ### Additional comments

Reviewers please ensure that the badges works as intended - this has net been tested.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,53 +1,43 @@
-[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0)
-# BHoM
 
-A great place to start is reading our Wiki [here](https://github.com/BHoM/documentation/wiki) including pages like the [Structure of the BHoM](https://github.com/BHoM/documentation/wiki/Structure-of-the-BHoM) and [Using the BHoM](https://github.com/BHoM/documentation/wiki/Using-the-BHoM).
+[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0) [![Build status](https://ci.appveyor.com/api/projects/status/ja0l9sovnk48cf31/branch/master?svg=true)](https://ci.appveyor.com/api/projects/status/rfem_toolkit/branch/master) [![Build Status](https://dev.azure.com/BHoMBot/BHoM/_apis/build/status/RFEM_Toolkit/RFEM_Toolkit.CheckCore?branchName=master)](https://dev.azure.com/BHoMBot/BHoM/_build/latest?definitionId=93&branchName=master)
```
Comment (FraserGreenroyd) [inline README.md:2]: `DefinitionId` should be `186` for the last badge

File: README.md [hunk 2]
```diff
@@ -1,53 +1,43 @@
...
 
-### Clone and build the Core BHoM Repos
+Dlubal RFEM 5.19
 
-In the following build order:
-- [BHoM](https://github.com/BHoM/BHoM)
-- [BHoM_Engine](https://github.com/BHoM/BHoM_Engine)
-- [BHoM_Adapter](https://github.com/BHoM/BHoM_Adapter)
-- [BHoM_UI](https://github.com/BHoM/BHoM_UI)
+Dlubal RFEM 5.17
```
Comment (FraserGreenroyd) [inline README.md:18]: Can these be in order the other way round?

File: README.md (hunk 1 above)
Comment (MartinHenriksen) [reply README.md:2]: 👍 Thanks

File: README.md (hunk 2 above)
Comment (MartinHenriksen) [reply README.md:18]: If you prefer, sure 😃

**Pull request merged**